www/secmail/views/forms/icla.js.rb

```diff
@@ -89,6 +89,7 @@ def componentDidMount()
     end
 
     @realname = name
+    @pubname = name
```

Comment:
Case 1: The current behavior is to fill the real name from the email name. Then when I tab to public name, it fills from real name. This works most of the time when the real name == public name.

Case 2: But when real name and public name differ, I'd like to fill both real name and public name from email name and then when I fill real name from the document and tab to public name, real name is what's on the document and public name is from email name.

Case 3: And when JustTryingToMakeYourLife@difficult.org sends in 15 iclas, after I burst and staple 15 documents, I then process each one individually. I have to enter real name and re-enter the same into public name. How to make that easier? 

If we can tell that we have a burst/stapled document, we should not pre-fill either real name or public name. I tab to real name, fill it, tab to public name which should pre-fill from real name. How about a (clear form) button between ICLA and Real Name?

Case 4: FAX documents are handled like Case 3 with a clear form.

The onFocus code should remain. This is useful for Case 3 and 4 where I have to enter real name because there is no email name. I have to enter real name and then when I tab, public name is filled from real name.